### ISSUE TITLE
Add support for `pytorch>=1.0`

### ISSUE THREAD

**Issue by arpanmangal** [arpanmangal/CovidAID#2]
Present code requires `pytorch0.3` and `CUDA<=8.0`. The code needs to be updated to support `pytorch>1.0`. The main reason for using `pytorch0.3` is compatibility with trained CheXNet model.

**Comment by arpanmangal** [comment]
Continuing discussing from #1 

----------------
@SuryaKalia FYI

**Comment by arpanmangal** [comment]
@bartmch
Yes keeping `num_batches_tracked` as the default value of zero tensors is the best which could be done imo.
Further the keys not only differ in a `module.` string, but are also renamed like one has `...conv1`, while other version has `...conv.1`. Print the keys and compare to see all such variations.

**Comment by Mayur6999** [comment]
transfer.py code is not working plz check ASAP
There is a problem loading pytorch 0.3.0 and torchvision 0.2.0 plz add support
